### PR TITLE
:steam_locomotive: Use single route for all results

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Got rid of the 2 results routes in favour or a single one.

Started to move to use `res.locals` rather than `req.<thing>`

Next iteration of `services.js` is to move logic out and into libs

### CODE REVIEW

File: config/routes.js [hunk 1]
```diff
@@ -28,17 +28,8 @@ router.get('/search',
...
-  validateLocation,
-  urlUtils.urlForPharmacyPostcodeSearch,
-  servicesMiddleware.getPharmacies,
-  servicesMiddleware.getPharmacyOpeningTimes,
-  servicesMiddleware.prepareOpenThingsForRender,
-  servicesMiddleware.renderServiceResults
-);
-
 router.get('/results',
+  validateLocation,
```
Comment (neilbmclaughlin) [inline config/routes.js]: I'm still able to navigate to http://localhost:3000/results-open by virtue of the `route.get('/:view',` line in config/routes. results-open.nunjuck needs to be deleted.

**Comment by st3v3nhunt** [comment]
Fixes #62

File: app/middleware/services.js [hunk 2]
```diff
@@ -152,34 +153,43 @@ function getDisplayValuesMapper(location) {
...
+    altResultsMessage = 'See only open places nearby';
+
+    const tenClosestPlaces = req.pharmacyList
+          .sort(sortByDistance)
+          .slice(0, 10);
+
+    serviceList =
+      pharmacyMapper(tenClosestPlaces)
+        .map(getDisplayValuesMapper(location))
+        .sort(sortByDistanceInKms);
```
Comment (st3v3nhunt) [inline app/middleware/services.js]: TBH, I'd not looked at the detail in this change, I was focussing on moving stuff around to get rid of the 2 methods. Although `prepareOpenThingsForRender` had only a single sort `prepareForRender` still had 2 which is why this branch has...

Is it still required? Not that I can see.

Do you want it removing as part of this or the next refactor?

File: config/routes.js (hunk 1 above)
Comment (st3v3nhunt) [inline config/routes.js]: I'll get that gone

File: app/middleware/services.js (hunk 2 above)
Comment (neilbmclaughlin) [inline app/middleware/services.js]: It's easier to remove it than log it in an issue.

**Pull request merged**